test1.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
move to scripts/pipeline
rename to basic_page_verification_test.py or similar

Reply:
Sure